.github/workflows/release-amd64.yaml

```diff
@@ -5,6 +5,10 @@ on:
       target-arch:
         required: true
         type: string
+      release-type:
+        required: false
```

Comment:
If release-type isn't given won't we just error in `next-release-version` function now that we are not defaulting to minor? If there another reason I'm missing that it's not required?

Reply:
> If release-type isn't given won't we just error in `next-release-version` function now that we are not defaulting to minor? If there another reason I'm missing that it's not required?

Does `workflow_call` actually do something with `required`  ? I don't find it in https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions .

Reply:
This was a c&p error, fixed.

Also, I've tried to reduce the amount of info we add that's not that much useful for the ones consuming the release.  Hopefully with this we can have the new contributors for the next release (which is basically 5~6 releases into a single one).